custom.scss

```diff
@@ -2,6 +2,25 @@
 
 /*-- scss:rules --*/
 
+$module-bg-color: rgba(188, 188, 254, 0.25);
```

Comment:
This new transparent background color displays better in both light and dark mode.